tests/test_sort.py

```diff
@@ -62,7 +62,7 @@ def test_argsort_ndarray(self, dtype, axis):
         expected = np_array.argsort(axis=axis)
         assert_dtype_allclose(result, expected)
 
-    @pytest.mark.parametrize("kind", [None, "stable"])
+    @pytest.mark.parametrize("kind", [None, "stable", "mergesort", "radixsort"])
```

Comment:
As I know `kind=None` results into different algorithms, so I think it might be helpful to parametrize dtype in that test also (the same for `sort`)